src/polysolve/nonlinear/Solver.cpp

```diff
@@ -196,35 +196,32 @@ namespace polysolve::nonlinear
                    spdlog::logger &logger)
         : m_logger(logger), characteristic_length(characteristic_length)
     {
-        TCriteria criteria = TCriteria::defaults();
-        criteria.xDelta = solver_params["x_delta"];
-        criteria.fDelta = solver_params["advanced"]["f_delta"];
-        criteria.gradNorm = solver_params["grad_norm"];
-
-        criteria.xDelta *= characteristic_length;
-        criteria.fDelta *= characteristic_length;
-        criteria.gradNorm *= characteristic_length;
-
-        criteria.iterations = solver_params["max_iterations"];
-        // criteria.condition = solver_params["condition"];
-        this->setStopCriteria(criteria);
-
-        use_grad_norm_tol = solver_params["line_search"]["use_grad_norm_tol"];
```

Comment:
use_grad_norm_tol is missing?

Reply:
No, I just set the one in m_line_search directly rather than having a copy in Solver.

Reply:
See `void Solver::set_line_search(const json &params)`